brainscore_vision/models/temporal_model_AVID_CMA/model.py

```diff
@@ -20,7 +20,7 @@ def get_model(identifier):
         cfg_path = os.path.join(HOME, "configs/main/avid-cma/kinetics/InstX-N1024-PosW-N64-Top32.yaml")
         weight_path = load_weight_file(
             bucket="brainscore-vision",
-            relative_path="temporal_model_AVID-CMA/AVID-CMA_Kinetics_InstX-N1024-PosW-N64-Top32_checkpoint.pth.tar",
+            relative_path="temporal_model_AVID_CMA/AVID-CMA_Kinetics_InstX-N1024-PosW-N64-Top32_checkpoint.pth.tar",
```

Comment:
just double checking this is right -- You renamed the plugin directory in code **and** on S3?

Reply:
@deirdre-k